models/migrations/20190829125613_categories.js

```diff
@@ -0,0 +1,54 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('categories', tbl => {
+    tbl.increments();
+    tbl.text('architecture');
+    tbl.text('art');
+    tbl.text('biography');
+    tbl.text('body_mind_Spirit');
+    tbl.text('computers');
+    tbl.text('cooking');
+    tbl.timestamp('created_at', { useTz: true });
```

Comment:
Long term it makes sense to have a bunch of columns with a "1" if it matches it.  I believe queries complete faster with integers than strings.  For today, since we're just shoving in a bunch of books from the outside that won't stay in our DB very long, I'd say let's just put an "Other" text category in there and we'll just put all of our books there with a name

Would also suggest a consistent naming scheme.  ie: `body_mind_Spirit` would probably be best all lower case

Reply:
well we need at least three categories per request from Sid